### ISSUE TITLE
As a supervisor, I want to be able to delete/modify the sharing of an employee profile with another supervisor.

### ISSUE THREAD

**Issue by phoebesolomon** [bcgov/performance#269]

**Comment by phoebesolomon** [comment]
![Supervisor - Share Employee Profile 9.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/b369a377-bcb8-4cf3-9298-fd949bf2e0e0)![Supervisor - Share Employee Profile 10.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/258066b1-4b30-4718-93b1-470f63f27ad6)![Supervisor - Share Employee Profile 11.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/f0a58320-dc19-4cf0-9bd5-649123e190ff)![Supervisor - Share Employee Profile 12.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/8d7ccc65-8826-44f1-a182-647fa44b527f)![Supervisor - Share Employee Profile 13.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/53f9bcca-8044-4f91-b4ea-f533b2e1ae03)![Supervisor - Share Employee Profile 14.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/8049e243-7cfe-4df9-a826-178c747a3578)![Supervisor - Share Employee Profile 15.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/5e8ca00e-b240-4fb4-8598-432ba13ab0ba)![Supervisor - Share Employee Profile 16.jpg](https://images.zenhubusercontent.com/60621c7c61819dc090d375b2/1a2d934d-091e-463b-9146-180ec83904e1)

**Comment by Travis-A-Clark** [comment]
Just capturing a thought here: we will need to introduce logic that only allows someone to share the elements of a profile that they have access to. In other words, if supervisor 1 shares the goals section of employee 1 with supervisor 2, then supervisor 2 should be able to share that goals section with supervisor 3. However, since supervisor 2 does not have access to the conversations section for employee 1, then supervisor 2 cannot grant this access to anyone else.

Basically, you can pass on your own level of authorization, but cannot go beyond or create new authorization.
